versioned_docs/version-8.5/components/connectors/protocol/soap.md

```diff
@@ -0,0 +1,211 @@
+---
+id: soap
+title: SOAP Connector
+sidebar_label: SOAP Connector
+description: The SOAP Connector allows you to connect your BPMN with SOAP services.
+---
+
+**SOAP** is a messaging protocol specification for exchanging structured
+information in the implementation of web services in computer networks.
+
+The **SOAP Connector** allows you to interact with [SOAP](https://en.wikipedia.org/wiki/SOAP) services endpoints
+from your BPMN process.
+
+## Prerequisites
+
+The **SOAP Connector** is only supported by self-managed Camunda instances.
+
+To use the **SOAP Connector**, you need to have an active SOAP service.
+
+## Create a SOAP Connector task
+
+To use the **SOAP Connector** in your process, either change the type of existing task by clicking on it and using
```

Comment:
"either change the type of existing task" should be "either change the type of an existing task"?